### PR TITLE
PPSE-701: ability to handle multiple kafka hosts and short-circuit bootstrap servers as ip addresses

### PR DESCRIPTION
**Pull request by bypaik**
expanded config to use delimited list of broker hosts

### CODE REVIEW

**Review by calexicoz — APPROVED** [review]
LGTM. Just so I understand well, the goal of this is essentially to allow to provide more than 1 hostname to bootstrap IPs, and I'm guessing this is required by the move to MSK. Is that correct?

**Comment by bypaik** [comment]
> LGTM. Just so I understand well, the goal of this is essentially to allow to provide more than 1 hostname to bootstrap IPs, and I'm guessing this is required by the move to MSK. Is that correct?

current code sets bootstrap servers by their ip addresses.  that works for self-managed cluster where ip addresses are hardcoded or set by cloud-compose.  with MSK, there's no guarantee the IPs will remain consistent.  so this is my attempt at keeping backward-compatibility to set bootstrap servers by IP and adding:
- capability to set multiple bootstrap servers by domain separated by delimiters
- skipping setting bootstrap servers by IP addresses

i've thought of a few more changes needed.  i will add a follow up commit(s) soon.

File: src/main/java/com/washingtonpost/kafka/Configuration.java [hunk 1]
```diff
@@ -51,26 +69,61 @@ private Configuration() {
...
+                    this.findHosts(config.kafka.host).collect(Collectors.toList()), 
+                    config.kafka.port);
+            } else {
+                config.kafka.bootstrapServers = this.appendPort(config.kafka.host, config.kafka.port);
+            }
         }
     }
 
+    boolean useIp(final String value) {
+        return value == null || BooleanUtils.isTrue(BooleanUtils.toBooleanObject(value));
```
Comment (bypaik) [inline src/main/java/com/washingtonpost/kafka/Configuration.java]: leading `value == null` should change to add blank string as the java object for Kafka will initialize to empty string

File: src/main/java/com/washingtonpost/kafka/Configuration.java [hunk 2]
```diff
@@ -97,7 +150,9 @@ public KafkaProducer getKafkaProducer() {
         @JsonProperty("bootstrap.servers")
         public String bootstrapServers;
         public String host;
-        public String port;
+        public int port;
+        @JsonProperty("use.ip")
+        public String useIp;
```
Comment (bypaik) [inline src/main/java/com/washingtonpost/kafka/Configuration.java:155]: need add this to the config template file

**Pull request merged**